### DISCUSSION TITLE
Rental module

### DISCUSSION THREAD

**Discussion by blu-IT** [YetiForceCompany/YetiForceCRM#15406] [Feature improvement]
Hi,

I would love to see a module for rental. 
I have a customer who has different things to rent and I found no way to integrate such a service in YF. Maybe somebody has an idea, but even the internal module for reservations is not offering the functinallity I hoped.

Has someone here experience how to integrate the possibility of renting this to customers?

Thanks!

Regards
Ivan

**Comment by apcloic** [comment]
Hi, 
Maybe you can give more details regarding the functionalities you are looking for? 
Basically, if you have programming skills, you can do whatever you want with yeti.

**Reply by blu-IT** [reply]
Well, something like a inventory list of devices for rent and a free/busy calendar for devices. 
I have checked the module for services and service contracts and also products in YF, but have not found a way to manage inventory that can be rent and comes back into the stock.

"leihs" would be a good starting point for a feature check: https://github.com/leihs/leihs

**Reply by moemago03** [reply]
Hi, did you find a solution? 
I need needed that too.
Thanks.

**Reply by blu-IT** [reply]
@moemago03 
> Hi, did you find a solution? I need needed that too. Thanks.

Unfortunately not.

**Reply by apcloic** [reply]
@blu-IT 
You could try to write custom scripts/handler to deal with inventory I/O.
Also, you may also have a look at Reservations module which is designed for internal resources but it may give you some ideas.
https://gitdeveloper.yetiforce.com/index.php?module=Reservations&view=Calendar&mid=83&parent=76
